### PR TITLE
LIB-280 feat(navbar): basic component, stories, tests and docs

### PR DESCRIPTION
**Pull request by Trebh**
Jira:
https://fiscozen.atlassian.net/browse/LIB-280

### CODE REVIEW

File: packages/navbar/src/FzNavbar.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+})
+
+const isMobile = computed(() => Boolean(width.value <= 1024))
+const isVertical = computed(() => Boolean(props.variant === 'vertical'))
+const isHorizontal = computed(() => Boolean(props.variant === 'horizontal'))
+</script>
+
+<template>
+  <header
+    class="flex px-12 py-12 shadow"
```
Comment (oxcened) [inline packages/navbar/src/FzNavbar.vue:33]:
```suggestion
    class="flex p-12 shadow"
```

File: packages/navbar/src/FzNavbar.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+onMounted(() => {
+  window.addEventListener('resize', onResize)
+})
+onUnmounted(() => {
+  window.removeEventListener('resize', onResize)
+})
+
+const isMobile = computed(() => Boolean(width.value <= 1024))
+const isVertical = computed(() => Boolean(props.variant === 'vertical'))
+const isHorizontal = computed(() => Boolean(props.variant === 'horizontal'))
```
Comment (oxcened) [inline packages/navbar/src/FzNavbar.vue:28]: A che serve `Boolean()` qui?

File: packages/navbar/src/FzNavbar.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        <slot name="user-menu"></slot>
+      </div>
+    </template>
+    <template v-else>
+      <FzIconButton iconName="bars" variant="secondary" tooltip="menu" :disabled="false" />
+      <div>
+        <slot name="brand-logo" :isMobile></slot>
+      </div>
+      <div>
+        <slot :name="variant === 'horizontal' ? 'notifications' : 'user-menu'"></slot>
```
Comment (oxcened) [inline packages/navbar/src/FzNavbar.vue:61]:
```suggestion
        <slot :name="isHorizontal ? 'notifications' : 'user-menu'"></slot>
```

File: packages/navbar/src/types.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export type FzNavbarVariant = 'horizontal' | 'vertical'
+
+export interface FzNavbarProps {
+  /*
```
Comment (oxcened) [inline packages/navbar/src/types.ts:4]:
```suggestion
  /**
```

**Review by oxcened — APPROVED** [review]

**Review by panicofr — APPROVED** [review]

**Pull request merged**